web/src/components/storage/device-utils.jsx

```diff
@@ -224,38 +223,64 @@ const ItemContent = ({ device }) => {
 };
 
 /**
- * Component for selecting a storage device
+ * Component for listing storage devices.
  * @component
  *
  * @param {Object} props
- * @param {StorageDevice} [props.selected] - Default selected device, in any
- * @param {StorageDevice[]=[]} [props.devices] - Devices to show in the selector
- * @param {onSelectFn} [props.onSelect] - Function to be called when a device is selected
+ * @param {StorageDevice[]} props.devices - Devices to show.
+ */
+const DeviceList = ({ devices }) => {
```

Comment:
Note: we've DeviceList and DeviceSelector into device-utils file by now. Most probably we will look for a better place once we manage to see how useful can the ListBox be exposed under core namesapce.